apple/RNSVGUIKit.h

```diff
@@ -0,0 +1,41 @@
+// Most (if not all) of this file could probably go away once react-native-macos's version of RCTUIKit.h makes its way upstream.
+// https://github.com/microsoft/react-native-macos/issues/242
+
+#if !TARGET_OS_OSX
+
+#import <UIKit/UIKit.h>
+
+#define RNSVGColor UIColor
+#define RNSVGPlatformView UIView
+#define RNSVGTextView UILabel
+#define RNSVGView UIView
+
+#else // TARGET_OS_OSX [
+
+#import <React/RCTUIKit.h>
```

Comment:
This import can be shared and then we can drop the UIKit/UIKit.h import?

Reply:
Unfortunately, we can't do this until RCTUIKit.h becomes available in react-native.